### ISSUE TITLE
A git tag for univalence for citation

### ISSUE THREAD

**Issue by favonia** [RedPRL/sml-redprl#497]
@cangiuli I suppose you can tag the commit with any reasonable name you like ~after #496 is merged~? @jonsterling I wonder if there's any convention on tag names?

**Comment by favonia** [comment]
We can even use https://git.io/ after tagging the commit...

**Comment by cangiuli** [comment]
I'll let you or Jon decide what to call the tag. Your suggestion of "arxiv2017" seems fine to me.

**Comment by cangiuli** [comment]
Actually, hold on a sec, I'm opening another pull request in a minute.

**Comment by cangiuli** [comment]
It's called arxiv2017.
